CODEOWNERS

```diff
@@ -6,6 +6,7 @@
 #
 # Use git ls-files '<pattern>' without a / prefix to see the list of matching files.
 
+/engine/src/build/config/ios/** @jmagman
 /packages/flutter_tools/templates/module/ios/ @jmagman
 /packages/flutter_tools/templates/**/Podfile* @jmagman
 /packages/flutter_tools/lib/src/ios/ @jmagman
```

Comment:
@jmagman Should I transfer these to @vashworth?

Reply:
I'm probably not the right person to review these any more. 
@vashworth would you prefer these be put on you, or deleted?

Reply:
You can assign them to me. Thanks @loic-sharma

Reply:
Updated!